contracts/DecentralizedAutonomousTrust.sol

```diff
@@ -13,13 +13,17 @@ import "@openzeppelin/contracts-ethereum-package/contracts/utils/Address.sol";
 
 /**
  * @title Decentralized Autonomous Trust
- * This contract is the reference implementation provided by Fairmint for a
+ * This contract is the copy of the implementation provided by Fairmint for a
  * Decentralized Autonomous Trust as described in the continuous
  * organization whitepaper (https://github.com/c-org/whitepaper) and
- * specified here: https://github.com/fairmint/c-org/wiki. Use at your own
- * risk. If you have question or if you're looking for a ready-to-use
- * solution using this contract, you might be interested in Fairmint's
- * offering. Do not hesitate to get in touch with us: https://fairmint.co
+ * specified here: https://github.com/fairmint/c-org/wiki.
+ * Code from : https://github.com/Fairmint/c-org/blob/dfd3129f9bce8717406aba54d1f1888d8e253dbb/contracts/DecentralizedAutonomousTrust.sol
+ * Changes Added: https://github.com/Fairmint/c-org/commit/60bb63b9112a82996f275a75a87c28b1d73e3f11
+ *
+ * The code has been audited, and public audit results where published by Fairmint.
```

Comment:
It hasn't been officially published yet so we should hold off on this till we have a published link

Reply:
Yes, I am just assuming that by the time this will be in production the audit will be published, what do you think?

Reply:
removed.